### ISSUE TITLE
Homeassistant container script not working

### ISSUE THREAD

**Issue by Yonny24** [tteck/Proxmox#3]
Hi,

Great scripts here I must add first. Thank you.

Just having a little difficulty with the homeassistant container script.

I get a login on the console after starting the container and the ipaddress:8123 is not loading. 
IP is correct as I set a static IP for the container.

I assume this login is the root for the underlying linux but my root password does not work.

What suggestions do you have for me?
Do I need to access the config file to complete the set up?
/var/lib/docker/volumes/hass_config/_data
If I cannot log in to the console I can't access this.

Many thanks

![image](https://user-images.githubusercontent.com/64428686/141459711-925f533b-99e0-4779-a66c-7070c8722f12.png)

**Comment by tteck** [comment]
I just ran the script, I'm not having any issues myself. Try rebooting the LXC, if that doesn't work
I'd suggest that you run the script again, (there's one for PVE6 and PVE7) wait for the script to finish.

**Comment by Yonny24** [comment]
Thanks let me try again, I used the PVE7 as I have the latest. 
The error that came up just destroyed the container so had no chance of rebooting.

**Comment by Yonny24** [comment]
hmm odd, I tried rebooting many times. I used ve7 as I have the latest ve.
let me try again, I'll reboot the host also

**Comment by Yonny24** [comment]
yeahhh yes 2nd time after a host reboot - all working!!

Thanks so much

**Comment by tteck** [comment]
Good - Issue closed
